src/components/MediaSettings/MediaSettings.vue

```diff
@@ -25,12 +25,12 @@
 		size="small"
 		:container="$store.getters.getMainContainerSelector()"
 		@close="closeModal">
-		<div class="device-checker">
-			<h2 class="device-checker__title">
+		<div class="media-settings">
+			<h2 class="media-settings__title">
 				{{ t('spreed', 'Camera and microphone check') }}
 			</h2>
 			<!-- Preview -->
-			<div class="device-checker__preview">
+			<div class="media-settings__preview">
```

Comment:
We need to lock sizes of video component somewhere.
It keep jumping from one width to another, when you're toggling the camera on/off